js/apps/system/_admin/aardvark/APP/frontend/js/templates/userBarView.ejs

```diff
@@ -16,35 +18,15 @@
         <% } %>
         </p>
     </li>
-    <li class="dropdown-footer">
-      <button id="userLogout" class="button-primary btn-small pull-right">Logout</button>
-    </li>
-  </ul>
-
-<!--
-<ul class="navlist" id="userBarUl">
-
-  <li class="dropdown user-menu userImg">
-    <a href="#" class="tab userImg" id="user" >
-      <img class="user-menu-img" src="<%=img%>" id="userimage" /> <b class="caret"></b>
-    </a>
-    <ul class="user-dropdown-menu" id="user_dropdown">
-      <li class="dropdown-header" style="text-transform: none">
-        <%
-        if (name) {%>
-        <%=_.escape(name)%> (<%=_.escape(username)%>)
-        <% } else {%>
-        <%=_.escape(username)%>
-        <% } %>
-      <li class="dropdown-item">
-        <a id="userProfile" class="tab" href="#user">User Profile</a>
+    <% if (!ssoEnabled) {%>
+      <li class="dropdown-footer">
+        <button id="userLogout" class="button-primary btn-small pull-right">Logout</button>
       </li>
-      <li class="dropdown-item">
-        <a id="userLogout" class="tab">Logout</a>
+    <% }%>
+    <% if (ssoEnabled) {%>
+      <li class="dropdown-footer" style="max-width: 200px; line-height: 1.4; padding: 0 10px !important;">
+        To log out of this DB, log out of the ArangoGraph.
```

Comment:
Please change the message to "This login is automatically managed by ArangoGraph."